example/test.py

```diff
@@ -218,7 +214,7 @@ def test_nginx_access_logs(self):
             # Assert remote_addr is an IPv4 (roughly)
             'remote_addr': MatchesRegex(
                 r'^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}$'),
-            'http_host': Equals('127.0.0.1:{}'.format(self.web_port)),
+            'http_host': MatchesRegex(r'^127.0.0.1:\d{4,5}$'),
```

Comment:
Again, having an object we could call `.port` or something on would be nicer here.